### PR TITLE
fix: Missing files from PR #61

### PR DESCRIPTION
**Pull request by streino**
Missing files that should have gone into PR #61.

### CODE REVIEW

**Review by alexgarel — APPROVED** [review]
LGTM - thank you @streino 

I have a question though, and wait for your answer before merging.

File: lib/callback.py
Comment (streino) [reply lib/callback.py:19]: That code was extracted from [train.py](https://github.com/openfoodfacts/off-category-classification/blob/2.0/train.py#L78). I don't know exactly what it does but the way to use it is to add it to `model.fit(callbacks=...)` in place of the standard TensorBoard callback.

If you prefer I can leave it in `attic` until someone needs it.

Comment (alexgarel) [reply lib/callback.py:19]: yep, better move it to attic (as we already move a lot of stuff there :-D)

Comment (streino) [reply lib/callback.py:19]: Ok, so I'll just delete `lib/callback.py` since that code is already in `attic/train.py`. No need to have it twice.

**Pull request merged**
